<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>622</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-size: 1000px 670px;
}
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="BackgroundLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>1000</width>
       <height>670</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/Background3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="OTPLoginWidget">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>341</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="LoginPage">
      <widget class="QFrame" name="Login">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>341</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#Login {
    background-color: rgba(255, 255, 255, 0.13);
    border-radius: 24px;
}
</string>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <widget class="QLabel" name="titleLabel">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel {
    font-size: 24px;
    font-weight: bold;
	color: rgb(218, 224, 230);
}
</string>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="idLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>105</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idLineEdit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>261</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="pwLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>195</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwLineEdit">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>261</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="SubmitButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>330</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #f37321;
    color: white;
    border-radius: 6px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>SUBMIT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SignUpButton">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>280</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>
QPushButton {
    background: transparent;
    color: #bbb;
    border: none;
    text-decoration: underline;
}
</string>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="OTPLoginPage">
      <widget class="QFrame" name="OTPLogin">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>341</width>
         <height>421</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
#OTPLogin {
    background-color: rgba(255, 255, 255, 0.13);
    border-radius: 24px;
}
</string>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <widget class="QLabel" name="titleLabel_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel {
    font-size: 24px;
    font-weight: bold;
	color: rgb(218, 224, 230);
}
</string>
        </property>
        <property name="text">
         <string>LOGIN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="idLabel_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>155</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>OTP 인증번호 / 복구 번호</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idLineEdit_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>261</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #FFFFFF</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="placeholderText">
         <string>공백 없이 입력해주세요</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SubmitButton_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>330</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #f37321;
    color: white;
    border-radius: 6px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>SUBMIT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SignUpButton_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>280</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>
QPushButton {
    background: transparent;
    color: #bbb;
    border: none;
    text-decoration: underline;
}
</string>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>50</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>CCTV Monitoring System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    color: #333;
}</string>
    </property>
    <widget class="QLabel" name="BackgroundLabel_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>1000</width>
       <height>670</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/SIGNUP_.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="OTPLoginWidget_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>331</width>
       <height>501</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="SignUpPage">
      <widget class="QFrame" name="SignUp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>501</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
#SignUp {
    background-color: rgba(255, 255, 255, 200);
    border-radius: 24px;
}

</string>
       </property>
       <widget class="QLabel" name="titleLabel_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel {
    font-size: 24px;
    font-weight: bold;
	color: #686971;
}
</string>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
       <widget class="QLabel" name="idLabel_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>105</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(104, 109, 127);</string>
        </property>
        <property name="text">
         <string>ID 입력</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="IDLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>251</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">IDLabel{
	background-color: white;
    color: black;
    border-radius: 6px;
    padding: 6px;
    border: 1px solid #ccc;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="pwLabel_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>195</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(104, 109, 127);</string>
        </property>
        <property name="text">
         <string>Password 입력</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwLineEdit_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>251</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="SubmitButton_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>410</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #f37321;
    color: white;
    border-radius: 6px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>SUBMIT</string>
        </property>
       </widget>
       <widget class="QLabel" name="pwLabel_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>285</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(104, 109, 127);</string>
        </property>
        <property name="text">
         <string>Password 확인</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwLineEdit_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>251</width>
          <height>40</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>360</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: rgb(104, 109, 127);
    spacing: 5px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 1px solid #ccc;
    background: white;
}

QCheckBox::indicator:checked {
    image: none;
    background-color: #f26b1d; /* 체크 시 배경색 */
}

QCheckBox::indicator:unchecked {
    image: none;
    background-color: white;
    border: 1px solid #ccc;
}</string>
        </property>
        <property name="text">
         <string>2단계 인증 사용(OTP)</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="OTPSignUpPage">
      <widget class="QFrame" name="OTPSignUp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>501</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
#OTPSignUp {
    background-color: rgba(255, 255, 255, 200);
    border-radius: 24px;
}
</string>
       </property>
       <widget class="QLabel" name="titleLabel_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>101</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
QLabel {
    font-size: 24px;
    font-weight: bold;
	color: #686971;
}
</string>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
       <widget class="QPushButton" name="SubmitButton_6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>400</y>
          <width>120</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #f37321;
    color: white;
    border-radius: 6px;
    font-weight: bold;
}
</string>
        </property>
        <property name="text">
         <string>확인</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>110</y>
          <width>211</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>340</y>
          <width>211</width>
          <height>26</height>
         </rect>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
